practitioner-service/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I know it is not part of this story - but I think we need to have 1 checkstyle.xml and then let everyone use it and run it as part of the CI @itye-msft  did you open a story for that? Should I add it?

Reply:
this doesn't yet work on main branch and will be implemented as part of the next PR.